modules/tools/00_mod.mk

```diff
@@ -69,6 +69,7 @@ tools += kubectl=v1.33.3
 # renovate: datasource=github-releases packageName=kubernetes-sigs/kind
 tools += kind=v0.29.0
 # https://www.vaultproject.io/downloads
+# renovate: datasource=docker packageName=hashicorp/vault
```

Comment:
I think we should use datasource=go here too.

Reply:
I think this requires more careful thinking. Should we just postpone this change to a follow-up PR and get this merged?